arangod/RocksDBEngine/RocksDBPrimaryIndex.cpp

```diff
@@ -156,7 +156,8 @@ class RocksDBPrimaryIndexEqIterator final : public IndexIterator {
     LocalDocumentId documentId =
         _index->lookupKey(_trx, arangodb::velocypack::StringRef(_key->slice()), canReadOwnWrites());
     if (documentId.isSet()) {
-      cb(documentId, _key->slice());
+      auto data = SliceArrayCoveringData(_key->slice());
```

Comment:
Do we really need `SliceArrayCoveringData` here?
The primary index does not store VPack arrays, but stores the indexed `_key` values as simple strings (not even vpack).

Reply:
renamed to SliceCoveringData to exclude misleading Array from name